pyproject.toml

```
@@ -36,6 +36,7 @@ dependencies = [
     "importlib_resources==6.4.0",
     "docstring_parser==0.16", # CLI help-formatter
     "rich_argparse==1.4.0", # CLI help-formatter
+    "setuptools==69.5.1",
```

how about limiting like this?
```suggestion
    "setuptools<70",
```

---

Thanks, Done -  https://github.com/openvinotoolkit/training_extensions/pull/3609/commits/f924ead7e36a3ee0f555343abb6ec0c2aea961e9